### PR TITLE
Feature/kd8 w 104/db insert debug

### PR DESCRIPTION
**Pull request by ChadRehmKineticData**
Added an enable_trace_logging info value, removed logging of submission values, updated the sql logger to log to the Task engine log.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -64,3 +64,7 @@ Kinetic Platform [handlers] (2021-10-30)
 Kinetic Platform [handlers (2022-04-01)
   * [kinetic_request_ce_submission_db_insert_v1]
     * PER-268 fixed unique key constraint bug.
+
+Kinetic Platform [handlers (2022-08-15)
+  * [kinetic_request_ce_submission_db_insert_v1]
+    * PER-268 added info value to db insert handler.
```
Comment (jackboespflug) [inline changelog.md:70]: KD8W-104

File: handlers/kinetic_request_ce_submission_db_insert_v1/handler/init.rb [hunk 2]
```diff
@@ -160,7 +159,7 @@ def initialize(input)
     
     # Output SQL statements if the 'trace' level info parameter is set to true.
     @db.sql_log_level = :debug if @enable_trace_logging
-    @db.logger = Logger.new($stdout) if @enable_debug_logging
+    @db.logger = org.apache.log4j.LogManager.getLogger("com.kineticdata") if @enable_trace_logging
```
Comment (jackboespflug) [inline handlers/kinetic_request_ce_submission_db_insert_v1/handler/init.rb:162]: Do we need to check if running on Task 4 and adjust the log4j package name? I seem to remember the package name changed between log4j 1.2 (task 4) and log4j 2.0 (task 5).

**Review by jackboespflug — CHANGES_REQUESTED** [review]

File: handlers/kinetic_request_ce_submission_db_insert_v1/handler/init.rb (hunk 2 above)
Comment (ChadRehmKineticData) [reply handlers/kinetic_request_ce_submission_db_insert_v1/handler/init.rb:162]: After testing `org.apache.log4j.LogManager.getLogger("com.kineticdata")` in Task v5.2.8 and v4.5.0-SNAPSHOT it appears that the line of code is supported across many versions of Task.  Should there still be a change made?

File: changelog.md (hunk 1 above)
Comment (ChadRehmKineticData) [reply changelog.md]: I can confirm by looking at the changed file for this PR that the ticket number has been updated.

**Review by jackboespflug — APPROVED** [review]

**Pull request merged**